telemetry/specs/INT.mdk

```diff
@@ -734,14 +734,14 @@ both metadata (switch id and queue occupancy).
 * INT type value for the hop-by-hop INT Header type (i.e., the one that
 intermediate switches must process) is 1, and that the type value for the
 destination INT Header is 2.
+* The maximum number of hops (network diameter) is 8.
 * The INT metadata header uses the following field values in the metadata header,
 unless specified otherwise in each example.
   - Ver = 0
   - Rep = 0 (No replication)
   - C = 0
   - E = 0 (Max Hop Count not exceeded)
   - Instruction Count = 2 (for switch id & queue occupancy)
```

Comment:
May be say here that the INT source starts with a remaining hop count = 8

Reply:
Sorry, noticed that you do say maximum number of hops (network diameter) is 8. Hence INT source starts with a remaining hop count of 8.